src/components/news-list-item/news-list-item.js

```diff
@@ -5,31 +5,32 @@ import { getNews } from "../../services/news-service";
 import Spinner from "../spinner/spinner";
 
 const NewsListItem = ({ newsID }) => {
-  const [news, setnews] = useState(null);
-  const [loading, setloading] = useState(false);
+  const [news, setNews] = useState(null);
+  const [loading, setLoading] = useState(true);
 
   useEffect(() => {
     let cancelled = false;
     let controller;
     let signal;
 
-    if (!cancelled) {
-      controller = new AbortController();
-      signal = controller.signal;
-      getNews(newsID, signal).then((news) => {
-        // console.log(news);
-        setnews(news);
-        setloading(true);
-      });
-    }
+    controller = new AbortController();
+    signal = controller.signal;
+    getNews(newsID, signal).then((news) => {
```

Comment:
I don't see the need to store signal variable, you can use controller ref to get the value:

```
getNews(newsID, controller.signal)...
```